public/javascripts/SVValidate/src/keyboard/Keyboard.js

```diff
@@ -8,6 +8,41 @@ function Keyboard(menuUI) {
         addingComment: false
     };
 
+    // Add keydown listeners to the text boxes because escape
+    // key press is not being recognized when selected input text.
+    menuUI.optionalCommentTextBox.on('keydown', function (e) {
```

Comment:
Since all three of these are identical, can we rewrite this in a way where the function is only defined once?